### PR TITLE
Update Org Lookup as per UXA 

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* /bcgov/entity#12837

*Description of changes:*

- More UXA updates

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
Looks good.

File: ppr-ui/src/components/mhrTransfers/BusinessSearchAutocomplete.vue [hunk 1]
```diff
@@ -38,27 +43,25 @@
...
-          </v-list-item>
-        </v-list>
+        <div
+          v-else-if="
+            !isSearchResultSelected &&
+              autoCompleteIsActive &&
+              !searching &&
+              autoCompleteResults &&
+              autoCompleteResults.length === 0
+          "
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/BusinessSearchAutocomplete.vue]: Not required but a computed value might make this a cleaner template :)

**Review by cameron-eyds — APPROVED** [review]
Looks good to me, just a minor comment 👍

File: ppr-ui/src/components/mhrTransfers/BusinessSearchAutocomplete.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/mhrTransfers/BusinessSearchAutocomplete.vue]: Good idea. Will update.

**Pull request merged**